Libplanet/Blocks/Block.cs

```diff
@@ -21,6 +21,11 @@ namespace Libplanet.Blocks
     public class Block<T>
         where T : IAction, new()
     {
+        /// <summary>
+        /// The most latest protocol version.
+        /// </summary>
+        public const int CurrentProtocolVersion = 1;
```

Comment:
Latest protocol version is okay with constant?

Reply:
Yeah, this shouldn't be changed by applications, since it's for Libplanet-level protocol versioning.

Reply:
Wording "current" is confusing to me. What this constant means?

Reply:
It's the latest protocol version that this revision of Libplanet is aware of.

Reply:
Understood. Thx :)